src/core/credentials/transport/tls/tls_security_connector.h

```diff
@@ -275,6 +291,8 @@ class TlsServerSecurityConnector final : public grpc_server_security_connector {
   std::optional<absl::string_view> pem_root_certs_ ABSL_GUARDED_BY(mu_);
   std::optional<PemKeyCertPairList> pem_key_cert_pair_list_
       ABSL_GUARDED_BY(mu_);
+  std::optional<std::shared_ptr<SpiffeBundleMap>> spiffe_bundle_map_
```

Comment:
We should combine the `pem_root_certs_` and `spiffe_bundle_map_` fields using the same `std::shared_ptr<RootCertInfo>` type I suggested elsewhere.